### DISCUSSION TITLE
ReadMe info request

### DISCUSSION THREAD

**Discussion by gorg2331** [mebs/CM4-NAS#39] [Ideas]
Hi Mehdi! Superb project, looking to build one when summer is over. I created this thread so people can request some info added to the ReadMe that they would find useful. 

I think you could add the dimensions of the board in mm, since the size of this board is a Huge+ in my opinion. I was searching for a really portable CM4 NAS with one NVME drive as storage. Just to make it ultra slim. I know it will not be able to use the NVME to its full potential, but still for small factor it will work perfectly I think.

**Comment by mebs** [comment]
Hi @VitanovG !

The board is 88.5mm x 72.5mm. If you don't want to open the whole KiCad project, there is a .dxf file at the root that will give you the outline and hole placement.
Your use case sounds cool, but I think there are better suited boards out there. This one was not optimized to be slim (standing USB port, normal PCE Gen2 socket, all IO is upright...). By the time you add a board for your NVMe drive it will be quite a bit taller.
Feel free to branch from my design and update it by adding an NVMe socket to the back, but you could also look at existing boards.
Check out this one: https://hackaday.io/project/183430-bitpirat-computer-gen20

More boards: https://pipci.jeffgeerling.com/boards_cm

Good luck!

**Reply by gorg2331** [reply]
Thank you for the info! This bitpirat looks maybe more suited to my use case indeed! I know jeff, but never saw this bitrat project so far!
